packages/good-design/src/core/buttons/ClaimButton.tsx

```diff
@@ -34,221 +22,57 @@ const ClaimButton = ({
   ...props
 }: FVFlowProps) => {
   const { account } = useEthers();
-  const { Modal: FinalizationModal, showModal: showFinalizationModal } = useModal();
-  const { Modal: ActionModal, showModal: showActionModal, hideModal: hideActionModal } = useModal();
   const [claimConfirming, setClaimConfirming] = useState<boolean | undefined>(undefined);
   const [whitelistLoading, setWhitelistLoading] = useState(false);
-
-  const { loading, verify } = useFVModalAction({
-    firstName,
-    method,
-    chainId,
-    onClose: hideActionModal,
-    redirectUrl
-  });
+  const [hasStartedFv, sethasStartedFv] = useState(false);
```

Comment:
```suggestion
  const [faceVerifying, setFaceVerifying] = useState(false);
```
+ update rest of the code ofc

Reply:
done